packages/astro/src/runtime/server/render/util.ts

```diff
@@ -149,6 +149,46 @@ export function renderElement(
 	return `<${name}${internalSpreadAttributes(props, shouldEscape)}>${children}</${name}>`;
 }
 
+const noop = () => {};
+class BufferedRenderer implements RenderDestination {
+	private __chunks: RenderDestinationChunk[] = [];
+	private __renderPromise: Promise<void> | void;
+	private __destination?: RenderDestination;
```

Comment:
tiny nit, but is there a reason to prefix them with `__`? I think TypeScript already guarantees private fields to not be accessible.

Reply:
Just habit really, it is convention in most of the repos I work in and eslint is often configured to pick up on it

If you prefer different, I'm happy to drop the prefix 👍

Reply:
Yeah I prefer without the prefix so it's consistent with the codebase, e.g.

https://github.com/withastro/astro/blob/b3028caecf1a96310e2ee7a65075a81c231d87be/packages/astro/src/runtime/server/transition.ts#L130-L132

https://github.com/withastro/astro/blob/b3028caecf1a96310e2ee7a65075a81c231d87be/packages/astro/src/runtime/server/render/astro/instance.ts#L15-L22